### PR TITLE
140 truncate descriptions

### LINKED ISSUE #140: Truncate descriptions on all items

### LINKED ISSUE #149: change link in admin order show
the item link should link to the admin item path

### PR DESCRIPTION
**Pull request by meyerhoferc**
closes #140 
closes #149 

* adds truncation and styling to admin items index
* adds redirect to root in 'WWW' for navbar
* updates some tests

### CODE REVIEW

File: app/views/admin/items/index.html.erb [hunk 1]
```diff
@@ -10,15 +10,15 @@
   </tr>
   <% @items.each do |item| %>
     <tr>
-      <td class="avatar">
+      <td class="avatar" id="admin-item-image">
```
Comment (CjMoore) [inline app/views/admin/items/index.html.erb:13]: avatars!

File: spec/features/admin/orders/admin_views_an_order_spec.rb [hunk 2]
```diff
@@ -57,6 +57,6 @@
     end
 
     click_on "#{item1.title}"
-    expect(current_path).to eq(item_path(item1))
+    expect(current_path).to eq(admin_item_path(item1))
```
Comment (CjMoore) [inline spec/features/admin/orders/admin_views_an_order_spec.rb:60]: 👍

**Review by CjMoore — APPROVED** [review]
Looks good ready for second review. @valerietrudell @cmacaulay

File: app/views/admin/items/index.html.erb (hunk 1 above)
Comment (cmacaulay) [reply app/views/admin/items/index.html.erb:13]: 🎉

File: app/views/admin/orders/show.html.erb [hunk 3]
```diff
@@ -14,7 +14,7 @@
       <tbody>
         <% @order.order_items.each do |order_item| %>
         <tr>
-          <td><%= link_to order_item.item.title, item_path(order_item.item) %></td>
+          <td><%= link_to order_item.item.title, admin_item_path(order_item.item) %></td>
           <td><%= number_to_currency(order_item.item.price, precision: 2) %></td>
```
Comment (cmacaulay) [inline app/views/admin/orders/show.html.erb:18]: 👍

**Pull request merged**